src/content/getting-started/install-an-application/_index.md

```diff
@@ -15,4 +15,238 @@ user_questions:
+
+After few seconds you will see the deployed status:
+
+```nohighlight
+kubectl get app test01-hello-world
+NAME                INSTALLED VERSION   CREATED AT   LAST DEPLOYED   STATUS
+test01-hello-world   2.3.2               11s          8s              deployed
+```
+
+Now you can access your workload cluster, check the app running, and see which ingress address has been generated:
+
+```nohighlight
+kubectl get ingress -n default
+NAME          CLASS   HOSTS                                        ADDRESS                                                                       PORTS     AGE
+hello-world   nginx   cluster.provider.k8s.giantswam.io   ab49484.cn-north-1.elb.amazonaws.com.cn   80, 443   2m29s
+```
+
+Sadly, the address isn't accessible because we've not passed the right domain on to the ingress controller.
+
+## Step 4: Configuring the right domain
+
+For every workload cluster there is a default wildcard record created automatically by our controllers. The domain is composed by the cluster name and the provider, the region and the base domain.
+
+```nohighlight
+kubectl get cm test01-cluster-values -ojsonpath="{.data.values}" | yq .baseDomain
+test01.capi.aws.k8s.gigantic.io
+```
+
+With that information we can customize our `hello-world` app to use the right domain. Checking the [app values schema](https://github.com/giantswarm/hello-world-app/blob/main/helm/hello-world/values.schema.json) we can see the `host` property should override in `hosts` and `tls` sections.
+
+Lets create a file with the values we want to override:
+
+```nohighlight
+cat > hello-world-values.yaml <<EOL
+ingress:
+hosts:
+- host: hello.test01.capi.pek.aws.k8s.adidas.com.cn
+  paths:
+  - path: /
+    pathType: Prefix
+tls:
+- secretName: hello-world-tls
+  hosts:
+  - hello.test01.capi.pek.aws.k8s.adidas.com.cn
+EOL
+```
+
+Now we can template the app again passing the file reference:
+
+```nohighlight
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=test01 \
+  --organization=giantswarm \
+  --name=test01-hello-world \
+  --target-namespace=default \
+  --user-configmap=hello-world-values.yaml \
+  --version=2.3.2 > hello-world.yaml
+```
+
+We push the file or apply the changes to the platform API and wait a couple of seconds to see the changes:
+
+```nohighlight
+kubectl get app test01-hello-world
+NAME                                  INSTALLED VERSION   CREATED AT   LAST DEPLOYED   STATUS
+test01-hello-world                     2.3.2               51m          8s             deployed
+```
+
+At this point you should be able to access the `hello-world` app using the domain you have configured. Check the `URL` value in the ingress resource has been updated to be sure the changes have been propagated. Now you should be able to see the hello world frontend.
+
+![Hello world page](hello-world.png)
```

Comment:
This should at best be consistent with our cluster naming (`test01`), even if you only just edit the address bar to fake it